UC-Berkeley-Extension/global
# Download files on install

## Description
This PR Should finally allow for the syncing of files from the dev-berkeleyglobal instance

## Testing instructions
1. composer install from local machine
2. `phing install` from virtual machine html directory

After doing this the files from the live site should be synched!
